datasets/noaa/_terraform/ghcnd_by_year_pipeline.tf

```diff
@@ -30,10 +30,10 @@ resource "google_bigquery_table" "lightning_strikes_by_year" {
   ]
 }
 
-output "bigquery_table-lightning_strikes_by_year-table_id" {
-  value = google_bigquery_table.lightning_strikes_by_year.table_id
+output "bigquery_table-ghcnd_by_year-table_id" {
+  value = google_bigquery_table.ghcnd_by_year.table_id
 }
 
-output "bigquery_table-lightning_strikes_by_year-id" {
-  value = google_bigquery_table.lightning_strikes_by_year.id
+output "bigquery_table-ghcnd_by_year-id" {
+  value = google_bigquery_table.ghcnd_by_year.id
```

Comment:
Why did these get renamed? Seems entirely different pipelines

Reply:
@adlersantos This is a terraform generated file.  I dont know why terraform did this.  If you know why, please let me know.  Thanks!